pkg/util.go

```diff
@@ -77,6 +77,30 @@ func IsTime(stringDate string) bool {
 	return false
 }
 
+func CreateGitIgnore(logger *zap.Logger, filePath string) error {
```

Comment:
I think this is a generic function so should be added to global utils instead of pkg utils.

Reply:
Added check for when gitignore file already exists

Reply:
Refactored to global utils